test/pylib/rest_client.py

```diff
@@ -78,3 +85,74 @@ async def put_json(self, resource: str, json: dict, host: str) \
         """Put JSON"""
         return await self.session.request(method="PUT", url=self._resource_uri(resource, host),
                                           json=json)
+
+    async def post(self, resource: str, params: Dict[str, str], host: str) \
+            -> aiohttp.ClientResponse:
+        """Post to remote resource or raise"""
+        resp = await self.session.post(self._resource_uri(resource, host), params=params)
+        if resp.status != 200:
+            text = await resp.text()
+            raise RuntimeError(f"status code: {resp.status}, body text: {text}, "
+                               f"resource {resource} params {params}")
+        return resp
+
+
+class ScyllaRESTAPIClient():
+    """Async Scylla REST API client"""
+
+    def __init__(self, port: int = 10000):
+        self.cli = TCPRESTClient(port)
+
+    async def stop(self):
+        """Close session"""
+        await self.cli.stop()
+
+    async def get_server_uuid(self, server_id: str):
```

Comment:
Rename this to `get_server_host_id`.

Reply:
Renamed to `get_host_id` in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d